File: Server/AiesecAppBackEnd/api/models/ComplainReply.js
/**
 * ComplainReply.js
 *
 * @description :: TODO: You might write a short summary of how this model works and what it represents here.
 * @docs        :: http://sailsjs.org/documentation/concepts/models-and-orm/models
 */

module.exports = {

  schema: true,
  autoCreatedAt: false,
  autoUpdatedAt: false,
  tableName: 'complainreply',
  identity: 'ComplainReply',

  attributes: {
    idComplainReply: {
      type: 'integer',
      primaryKey: true,
      unique: true,
      autoIncrement: true
    },

    Complain_idComplain: {
      model: 'Complain',
      required: true
    },

    User_idUser: {
      model: 'User',
      required: true
    },

    replyText: {
      type: 'string',
      required: true
    },

    createdDate: {
      type: 'datetime'
    },

    updatedDate: {
      type: 'datetime'
    },

    expired: {
      type: 'boolean',
      required: true,
      defaultsTo: false
    }
  }
};

File: Client/Aiesec-App/Aiesec_App/Models/ComplainReply.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Aiesec_App.Models
{
    public class ComplainReply : BaseDataObject
    {
        public int idComplainReply { get; set; }

        public int Complain_idComplain { get; set; }

        public int User_idUser { get; set; }

        public string replyText { get; set; }

        public string name { get; set; }

    }
}
